src/components/Pivot/Pivot.css

```diff
@@ -0,0 +1,29 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import 'src/css/variables/colors.css';
+@import 'src/css/variables/fonts.css';
+
+.y-pivot .ms-Pivot-link {
+  @extend text__small;
+  color: $color__deathMetal;
+  font-weight: $fontWeight__bold;
+  height: 28px;
```

Comment:
i'd make this 2.8rem so it grows if the user adjusts their browser's base font size